### ISSUE TITLE
mocha test runner failing

### ISSUE THREAD

**Issue by teyc** [alm-tools/alm#200]
Hi, I'm trying out the latest test feature on alm, and the console is reporting

```
MOCHA STDERR:   ...\node_modules\alm\src\server\workers\tested\runners\mochaInstrumenter.ts:9
import * as common from "./instrumenterCommon";     
^^^^^^                                              
SyntaxError: Unexpected token import                
    at Object.exports.runInThisContext (vm.js:76:16)            
```                                                

Error was reproduced on OS X as well as Windows 10, running Node 6.x.

```
$ npm list -g alm
`-- alm@2.0.2
$ typescript --init
$ typings init
$ typings install --global mocha
$ cat alm.json
{
  "tests": {
    "include": [
       "./tests/*.ts"
    ]
  }
}
$ cat tests/sample.ts                         
/// <reference path="../typings/index.d.ts" />
                                              
describe('simple test', () => {               
  it('adds 2 + 2', () => {                    
    throw 'bad stuff'                         
  })                                          
}) 
```

**Comment by basarat** [comment]
worked for me for some reason : 

![image](https://cloud.githubusercontent.com/assets/874898/18614582/24173550-7dd4-11e6-9bb9-f503a5bf3261.png)

![image](https://cloud.githubusercontent.com/assets/874898/18614587/3f881002-7dd4-11e6-8198-13b3a6d3e987.png)

**Comment by basarat** [comment]
Verified. Deployed version has this bug : 
![image](https://cloud.githubusercontent.com/assets/874898/18614759/68ad61ea-7dd8-11e6-9456-f8a6a700f0cc.png) 

Figuring out why :rose:

**Comment by teyc** [comment]
I'm new to alm, but happy to help out if you point to me where to start looking.

**Comment by basarat** [comment]
> I'm new to alm, but happy to help out if you point to me where to start looking.

I'll fix it. Things I've ruled out : `node_modules` folder doesn't fix it (basically copied my local copy to replace the npm installed copy and it still results in an error). Still since you asked :rose:

## How testing works 

First be familiar with how workers work : https://basarat.gitbooks.io/alm/content/contributing/async.html

Next its just this worker : https://github.com/alm-tools/alm/blob/master/src/server/workers/tested/testedWorker.ts

Which just calls `mochaRunner` : https://github.com/alm-tools/alm/blob/aa86da0eaa8afddda2b895b512ee3eaa2194ac29/src/server/workers/tested/testedWorker.ts#L173 

Which just executes this command to run mocha : https://github.com/alm-tools/alm/blob/aa86da0eaa8afddda2b895b512ee3eaa2194ac29/src/server/workers/tested/runners/mochaRunner.ts#L37-L99 

Based on the error it feels like `ts-node` isn't running (it should compile away the `import *` before handing it over to mocha :rose:

**Comment by teyc** [comment]
Thanks. Documenting this for someone like myself in the future - able to reproduce error executing

```
$ C:/Users/Chui/AppData/Roaming/npm/node_modules/alm/node_modules/mocha/bin/_mocha \
  --require C:/Users/Chui/AppData/Roaming/npm/node_modules/alm/node_modules/ts-node/register  \
  --require C:/Users/Chui/AppData/Roaming/npm/node_modules/alm/src/server/workers/tested/runners/mochaInstrumenter.ts \
  --reporter json --full-trace tests/sample.ts

```

**Comment by basarat** [comment]
@teyc thanks for the fix 🎉 🎈 ! What a great day to wake up to :heart: 

Released with `2.0.4` :rose:

![image](https://cloud.githubusercontent.com/assets/874898/18619898/f26f59b6-7e4a-11e6-8d20-e75e3f61c287.png)
